doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -211,7 +211,28 @@ By default, an alert agent will be called for node events, fencing events, and
 resource events. An agent may choose to ignore certain types of events, but
 there is still the overhead of calling it for those events. To eliminate that
 overhead, you may select which types of events the agent should receive.
-   
+
+Alert filters are configured within a ``select`` element inside an ``alert``
+element.
+
+.. list-table:: **Possible alert filters**
+   :class: longtable
+   :widths: 1 3
+   :header-rows: 1
+
+   * - Name
+     - Events alerted
+   * - select_nodes
+     - A node joins or leaves the cluster (whether at the cluster layer for
+       full nodes, or via a remote connection for Pacemaker Remote nodes).
+   * - select_fencing
+     - Fencing or unfencing of a node completes (whether successfully or not).
+   * - select_resources
+     - A resource action other than meta-data completes (whether successfully
+       or not).
+   * - select_attributes
+     - A transient attribute value is sent to the CIB.
```

Comment:
nitpick: may be clearer as "value update is sent to", "value is written to", "value write request is sent to", or something

Also may be good to document `<attribute>` sub-element explicitly instead of only via an example configuration below

Reply:
"update is sent" is most accurate. I'm not bothering with `<attribute>` since it's experimental and not entirely production-ready, and I don't want to encourage its use yet :)